Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Merge.swift

```diff
@@ -94,6 +99,8 @@ extension Docc {
                     guard fileManager.directoryExists(atPath: catalog.path) else {
                         throw ValidationError("No directory exists at '\(catalog.path)'")
                     }
+                    
+                    print("note: Using a custom landing page catalog isn't supported yet. Will synthesize a default landing page instead.")
```

Comment:
Why not remove this option for now?

Reply:
See [this comment](https://github.com/swiftlang/swift-docc/pull/1011#discussion_r1723704217)